sparkle/src/stories/Item.stories.tsx

```diff
@@ -79,61 +84,74 @@ export const ListItemExample = () => (
     <div className="s-flex s-flex-col s-gap-8">
```

Comment:
Why not killing this file, it's superseeded by `NewDropdown.stories.tsx`

Reply:
Well we still have examples of how to use Item, Item.Avatar, Item.Link and stuff

Reply:
Gracias!